packages/testsweets/lib/src/services/widget_capture_service.dart

```diff
@@ -9,8 +10,10 @@ class WidgetCaptureService {
 
   final _cloudFunctionsService = locator<CloudFunctionsService>();
 
+  /// Adding views key that we will add all the captured views to it cause
+  /// they doesn't have a viewName(views viewName is empry string)
```

Comment:
See comment in slack. We should set the viewName of the description equal to the view's name. We have to stick to our domain language to ensure no confusion when talking about these things.